--- ncbi-blast-2.9.0+-src/c++/src/algo/blast/core/blast_kappa.c
+++ ncbi-blast-2.9.0+-src_new/c++/src/algo/blast/core/blast_kappa.c
@@ -3413,12 +3413,11 @@
 
     Boolean interrupt = FALSE;
 #pragma omp parallel \
-    default(none) num_threads(actual_num_threads) \
+    num_threads(actual_num_threads) \
     if(actual_num_threads>1) \
     shared(interrupt, seqsrc_tld, score_params_tld, hit_params_tld, \
     gap_align_tld, results_tld, \
     redoneMatches_tld, \
-    STDERR_COMMA \
     numQueries, numMatches, theseMatches, \
     numFrames, program_number, subjectBlk_tld, positionBased, \
     default_db_genetic_code, localScalingFactor, queryInfo, \
--- ncbi-blast-2.9.0+-src/c++/src/app/blast/blast_app_util.cpp
+++ ncbi-blast-2.9.0+-src_new/c++/src/app/blast/blast_app_util.cpp
@@ -948,11 +948,7 @@
 
 void QueryBatchCleanup()
 {
-#if defined(NCBI_OS_LINUX) && HAVE_MALLOC_H
-	malloc_trim(0);
-#endif
     return;
-
 }
 
 END_NCBI_SCOPE
--- ncbi-blast-2.9.0+-src/c++/src/objtools/blast/seqdb_reader/test/seqdb_perf.cpp
+++ ncbi-blast-2.9.0+-src_new/c++/src/objtools/blast/seqdb_reader/test/seqdb_perf.cpp
@@ -138,7 +138,7 @@
     LOG_POST(Info << "Will go over " << oids2iterate.size() << " sequences");
 
     #pragma omp parallel default(none) num_threads(m_DbHandles.size()) \
-                         shared(oids2iterate) if(m_DbHandles.size() > 1)
+                         shared(oids2iterate, kScanUncompressed) if(m_DbHandles.size() > 1)
     {
         int thread_id = 0;
 #ifdef _OPENMP
