--- salmon-1.3.0_new/CMakeLists.txt
+++ salmon-1.3.0/CMakeLists.txt
@@ -345,11 +345,6 @@
   set(EXTRA_CMAKE_LIBRARY_FLAGS "-L${CMAKE_LIBRARY_PATH}")
 endif()

-find_package(Iconv REQUIRED)
-if(NOT Iconv_IS_BUILT_IN)
-  set(ICONV_LIB Iconv::Iconv)
-endif()
-
 find_package(LibLZMA)
 if(NOT LIBLZMA_FOUND)
   message("Will attempt to fetch and build liblzma")
--- salmon-1.3.0_new/include/BAMUtils.hpp
+++ salmon-1.3.0/include/BAMUtils.hpp
@@ -9,6 +9,7 @@
 }

 #include <unordered_map>
+#include <string>

 namespace salmon {
   namespace bam_utils {